src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/help/Get-AzureRmContainerRegistry.md

```diff
@@ -12,40 +12,38 @@ Gets a container registry.
 
 ## SYNTAX
 
-### ResourceGroupParameterSet
+### ListRegistriesParameterSet
 ```
-Get-AzureRmContainerRegistry [[-ResourceGroupName] <String>] [-DefaultProfile <IAzureContextContainer>]
- [<CommonParameters>]
+Get-AzureRmContainerRegistry [[-ResourceGroupName] <String>] [-IncludeDetail]
+ [-DefaultProfile <IAzureContextContainer>]
 ```
 
 ### RegistryNameParameterSet
 ```
-Get-AzureRmContainerRegistry [-ResourceGroupName] <String> [-Name] <String>
- [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+Get-AzureRmContainerRegistry [-ResourceGroupName] <String> [-Name] <String> [-IncludeDetail]
+ [-DefaultProfile <IAzureContextContainer>]
+```
+
+### ResourceIdParameterSet
+```
+Get-AzureRmContainerRegistry [-IncludeDetail] -ResourceId <String> [-DefaultProfile <IAzureContextContainer>]
 ```
 
 ## DESCRIPTION
-The **Get-AzureRmContainerRegistry** cmdlet gets a specified container registry or all the container registries in a resource group or the subscription.
+The Get-AzureRmContainerRegistry cmdlet gets a specified container registry or all the container registries in a resource group or the subscription.
 
 ## EXAMPLES
 
 ### Example 1: Get a specified container registry
 ```
 PS C:\>Get-AzureRmContainerRegistry -ResourceGroupName "MyResourceGroup" -Name "MyRegistry"
 
-Id                 : /subscriptions/3eb31d8d-2879-4706-89b4-4dc4047726c6/resourceGroups/MyResourceGroup/providers/Microsoft.ContainerRegistry/registries/MyRegistry
-ResourceGroupName  : MyResourceGroup
-Name               : MyRegistry
-Type               : Microsoft.ContainerRegistry/registries
-Location           : westus
-Tags               : {}
-SkuName            : Basic
-SkuTier            : Basic
-LoginServer        : myregistry.azurecr.io
-CreationDate       : 4/13/2017 3:48:57 PM
-ProvisioningState  : Succeeded
-AdminUserEnabled   : False
-StorageAccountName : myregistry154817
+   Container registry location: westus
+
+Registry Name        Sku        LoginServer                    CreationDate         Provisioni AdminUserE StorageAccountName
+                                                                                    ngState    nabled
+-------------        ---        -----------                    ------------         ---------- ---------- ------------------
+MyRegistry        Premium    MyRegistry.azurecr.io       10/31/2017 6:49:3... Succeeded  True
```

Comment:
The indention seems off. Can we improve?

Reply:
fixed